.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 ci:
-    skip: [ruff, pixi-update-default, pixi-update-cpu, pixi-update-gpu]
+    skip: [pixi-install-default, pixi-install-cpu, pixi-install-gpu]
```

Comment:
we no longer skip `ruff`

Reply:
I'm confused that it DID pass though! leave it on I guess... weird